src/renderer/services/db/api/repositories/account.ts

```diff
@@ -1,18 +1,39 @@
 import {Repository, Id} from "../repository";
 import {CGAccount} from "../../../../models/account";
-import {encodeKey} from "../../schema";
+import {encodeKey, Bucket} from "../../schema";
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+import {IDatabaseController} from "../../controller";
+import {Account} from "../../../../models/ssz/types";
+import {JSONSerializer} from "../../serializers/json";
 
 export class AccountRepository extends Repository<CGAccount> {
+    public constructor(config: IBeaconConfig, db: IDatabaseController) {
+        super(config, db, JSONSerializer, Bucket.account, Account);
+    }
+
     // Override get method to wrap deserialized data into CGAccount instance
     public async get(id: Id): Promise<CGAccount | null> {
         try {
             const value = await this.db.get(encodeKey(this.bucket, id));
             if (!value) return null;
-            const {name, directory, sendStats} = this.serializer.deserialize(value, this.type);
-
+            const {name, directory, sendStats} = this.serializer.deserialize(
+                value,
+                this.type
+            );
             return new CGAccount({name, directory, sendStats});
         } catch (e) {
             return null;
         }
     }
+
+    public async set(id: Id, value: CGAccount): Promise<void> {
+        const parsedAccount = {
+            name: value.name,
+            directory: value.directory,
+            sendStats: value.sendStats
+        };
+
+        //@ts-ignore
```

Comment:
You can remove this if you assign type to parsedAccount

Reply:
Actually we don't even need this override of the set method.